src/unity/schemas/IdentityOrganization.yml

```diff
@@ -0,0 +1,18 @@
+properties:
+  id:
+    type: number
+    description: quartz ID of the org
```

Comment:
Do we need `id` here? I don't think we want to confuse the UI with two IDs; I think we can safely isolate that bit of complexity to Quartz.

Reply:
We eliminated the `id` field here and in `IdentityUser.yml`. Just for some context, the `User.yml` has and `id` and an `idpeId`, so we kept the naming the same as that for consistency. In the future, it may be better to rename `idpeId` to just `id` and remove the Quartz ID from all instances of User & Org.

Reply:
Hmmm, that's unfortunate. I wonder if the UI uses that ID for anything.

While I do like the consistency you've gone for, I'm wondering if we should make this payload be what we want going forward and to a separate issue/PR to make the other usage consistent with this one.  Might be one less breaking change to the API that way when we fix this up later. Up to you, though.